lib/app/features/help_center/data/models/audio_model.dart

```diff
@@ -1,18 +1,23 @@
-import 'package:penhas/app/features/help_center/domain/entities/audio_entity.dart';
-import 'package:penhas/app/shared/logger/log.dart';
+import '../../domain/entities/audio_entity.dart';
+import '../../../../shared/logger/log.dart';
 
 class AudioModel {
-  static List<AudioEntity> fromJson(Map<String, dynamic>? json) {
-    if (json == null || json.isEmpty || json['rows'] == null) {
-      return List.empty();
-    }
+  const AudioModel({required this.audioList, required this.message});
 
+  factory AudioModel.fromJson(Map<String, dynamic>? json) {
+    final message = json!['message'] ?? '';
     final rows = json['rows'] as List<dynamic>;
```

Comment:
talvez usar a `?` para se acaso `json` vir nulo:
```suggestion
    final message = json?['message'] as String? ?? '';
    final rows = json?['rows'] as List<dynamic>? ?? [];
```

Reply:
Isso foi endereçado?